src/models/userModel.tsx

```diff
@@ -0,0 +1,6 @@
+type user = {
+  id: Number;
```

Comment:
define all this types with lowercase letter https://stackoverflow.com/questions/67155108/what-is-the-difference-between-number-and-number-in-typescript